src/sentry/sentry_metrics/use_case_id_registry.py

```diff
@@ -25,6 +27,12 @@ class UseCaseID(Enum):
     UseCaseKey.PERFORMANCE: UseCaseID.TRANSACTIONS,
 }
 
+USE_CASE_ID_WRITES_LIMIT_QUOTA_OPTION_NAME = {
```

Comment:
do we need per-usecase options? can't all usecases have the same quota settings, if not even share the same quotas? i think that's at least how cardinality limiter already works, right?

Reply:
I think the cardinality limiter is applying quotas by use case ID (via @ayirr7's changes):
https://github.com/getsentry/sentry/pull/49121/files#diff-5988eb8812e28f8a9178999665c1e1766f7f7bb263464cab41bbd08b789c8601R54-R57

Reply:
The idea is to be able to configure DB write limits on a per use case-basis, so I think that would mean having separate options for separate use cases (UseCaseID). For example, one use case (e.g. spans) might need to be tuned differently and might need separate write rate limits configured depending on its ingestion rate. We would also want to make sure that no single use case can overwhelm the metric pipeline, since multiple use cases share it.

Reply:
@untitaker does that make sense?

Reply:
that's fine. i think it would be easier to combine all usecases into a single option with a value like this:

```
{
"default": [<limit 1>],
"performance": [<limit 2>]
}
```

for a usecase, you fetch a single option, look up the usecase inside that option, and if there is no key for that usecase you fall back to the default key

otherwise for each new usecase, you have to configure limits explicitly

Reply:
That's a good idea

Reply:
What name do we want the default to be?

Reply:
Sure that default name makes sense to me. This is the one that any use case residing on the generic metrics pipeline would default to if it didn't have its own configured, right?

Reply:
yup!